blocklylib-core/src/main/java/com/google/blockly/android/AbstractBlocklyActivity.java

```diff
@@ -684,6 +698,23 @@ public void onClick(View v) {
     }
 
     /**
+     * This method finds and set the visibility of {@link R.id#blockly_zoom_in_button} and {@link R.id#blockly_zoom_out_button}
+     * from the view hierarchy according
+     * <p/>
+     * @param visible
+     */
+    protected void setScaleToolBoxVisibility(boolean visible){
```

Comment:
In matching the existing terminology of onConfigureZoom*() methods, rename this setZoomButtonVisibility().